Mathlib/GroupTheory/Index.lean

```diff
@@ -482,33 +552,38 @@ lemma _root_.AddSubgroup.index_toSubgroup {G : Type*} [AddGroup G] (H : AddSubgr
   rfl
 
 @[simp]
-lemma relindex_toAddSubgroup :
-    (Subgroup.toAddSubgroup H).relindex (Subgroup.toAddSubgroup K) = H.relindex K :=
+lemma relIndex_toAddSubgroup :
+    (Subgroup.toAddSubgroup H).relIndex (Subgroup.toAddSubgroup K) = H.relIndex K :=
   rfl
 
+@[deprecated (since := "2024-12-17")] alias relindex_toAddSubgroup := relIndex_toAddSubgroup
+
 @[simp]
-lemma _root_.AddSubgroup.relindex_toSubgroup {G : Type*} [AddGroup G] (H K : AddSubgroup G) :
-    (AddSubgroup.toSubgroup H).relindex (AddSubgroup.toSubgroup K) = H.relindex K :=
+lemma _root_.AddSubgroup.relIndex_toSubgroup {G : Type*} [AddGroup G] (H K : AddSubgroup G) :
+    (AddSubgroup.toSubgroup H).relIndex (AddSubgroup.toSubgroup K) = H.relIndex K :=
   rfl
 
+@[deprecated (since := "2024-12-17")]
+alias _root_.AddSubgroup.relindex_toSubgroup := _root_.AddSubgroup.relIndex_toSubgroup
+
 section FiniteIndex
 
 variable (H K)
 
 /-- Typeclass for finite index subgroups. -/
 class FiniteIndex : Prop where
   /-- The subgroup has finite index -/
-  finiteIndex : H.index ≠ 0
+  index_ne_zero : H.index ≠ 0
```

Comment:
Please add a deprecation for this and the one below. As @loefflerd  points out is is not possible to deprecate `{ finiteIndex := _ }` syntax, but you can at least keep `FiniteIndex.finiteIndex` around as an alias.